mesonbuild/compilers.py

```diff
@@ -2434,6 +2454,9 @@ def thread_link_flags(self):
     def get_option_link_args(self, options):
         return []
 
+    def get_linker_allow_undefined(self):
+        return []
```

Comment:
Ah, it's probably fine to leave it then, because it should never be called anyway (that's why I left it out of `ArLinker`.)